Makefile

```diff
@@ -2,178 +2,36 @@
 MKFILE_PATH := $(lastword $(MAKEFILE_LIST))
 CURRENT_DIR := $(patsubst %/,%,$(dir $(realpath $(MKFILE_PATH))))
 
-# Ensure GOPATH
-GOPATH ?= $(shell go env GOPATH)
-# assume last entry in GOPATH is home to project
-GOPATH := $(lastword $(subst :, ,${GOPATH}))
-
 # Tags specific for building
 GOTAGS ?=
 
-# Number of procs to use
-GOMAXPROCS ?= 4
-
 # Get the project metadata
-GO_DOCKER_VERSION ?= 1.13
-PROJECT := $(shell go list -m -mod=vendor)
 OWNER := "hashicorp"
-NAME := $(notdir $(PROJECT))
+NAME := "consul-template"
+PROJECT := $(shell go list -m)
 GIT_COMMIT ?= $(shell git rev-parse --short HEAD || echo release)
 VERSION := $(shell awk -F\" '/Version/ { print $$2; exit }' "${CURRENT_DIR}/version/version.go")
 
 # Current system information
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
 
-# Default os-arch combination to build
-XC_OS ?= darwin freebsd linux netbsd openbsd solaris windows
-XC_ARCH ?= 386 amd64 arm arm64
-# XC_EXCLUDE "arm64" entries excludes both arm and arm64
-XC_EXCLUDE ?= darwin/arm64 freebsd/arm64 netbsd/arm64 openbsd/arm64 solaris/arm64 windows/arm64 solaris/386
-
-# GPG Signing key (blank by default, means no GPG signing)
-GPG_KEY ?=
-
 # List of ldflags
 LD_FLAGS ?= \
 	-s \
 	-w \
-	-X ${PROJECT}/version.Name=${NAME} \
```

Comment:
makes sense to have `consul-template` hard-coded as the default value. Since it's a public variable, this options still stands for further customization for others 👍